conf/mesos-site.xml

```diff
@@ -19,54 +19,84 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
     <value>/var/run/hadoop-hdfs</value>
   </property>
-  
+
+  <property>
+    <name>mesos.hdfs.native-hadoop-binaries</name>
+    <description>Mark true if you have hadoop pre-installed on your host machines (otherwise it will be distributed by the scheduler)</description>
+    <value>false</value>
+  </property>
+
   <property>
     <name>mesos.hdfs.framework.mnt.path</name>
+    <description>Mount location (if mesos.hdfs.native-hadoop-binaries is marked false)</description>
     <value>/opt/mesosphere</value>
   </property>
 
   <property>
     <name>mesos.hdfs.state.zk</name>
-    <value>localhost:2181</value>
+    <description>Comma-separated hostname-port pairs of zookeeper node locations</description>
+    <value>master.mesos:2181</value>
```

Comment:
This isn't the mesos-dns config file, so please go back to the localhost default.